tests/integration/test_opensearch.py

```diff
@@ -82,9 +82,9 @@ async def test_deploy(ops_test: OpsTest, app_charm: PosixPath, data_integrator_c
             OPENSEARCH[ops_test.cloud_name],
             channel="edge",
             application_name=OPENSEARCH[ops_test.cloud_name],
-            num_units=3,
+            num_units=2,
         ),
-        ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="edge", config=tls_config),
+        ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="stable", config=tls_config),
```

Comment:
This is what Opensearch tests are using. (`edge` -- here the least was failing).

Reply:
**out-of-scope** I suppose we should find a "strategy" as DP. I would in general lean towards testing against an intermediate risk, like e.g. beta or candidate, and using tests outside of charm repos (like this one) to also gather information around possible regression or if next promotion (say beta to candidate, or candidate to stable) is reasonable. 

Anyhow, we don't yet have a strategy around this, so anything is good at this point!